File: PYTHON/Human.py
class Human:
    # private
    __nik = ""
    __nama =""
    __jenis_kelamin = ""
    
    def __init__(self, nik = "", nama = "", jenis_kelamin = ""):
        
        self.__nik = nik
        self.__nama = nama
        self.__jenis_kelamin = jenis_kelamin
        
    # get nik
    def get_nik(self):
        return self.__nik
    # set nik
    def set_nik(self, nik):
        self.__nik = nik
    # get nama
    def get_nama(self):
        return self.__nama
    # set nama
    def set_nama(self, nama):
        self.__nama = nama
    # get jenis_kelamin
    def get_jenis_kelamin(self):
        return self.__jenis_kelamin
    # set jenis_kelamin
    def set_jenis_kelamin(self, jenis_kelamin):
        self.__jenis_kelamin = jenis_kelamin
        
    

File: CPP/Human.cpp

#include "header.hh"

Human::Human() // kosntruktor kosong
{
    // default valuenya string
    this->nik = "";
    this->nama = "";
    this->jenis_kelamin = "";
}
Human::Human(string nik, string nama, string jenis_kelamin) // konstruktor parameter
{
    this->nik = nik;
    this->nama = nama;
    this->jenis_kelamin = jenis_kelamin;
}

string Human::get_nik()
{
    return this->nik;
}
void Human::set_nik(string nik)
{
    this->nik = nik;
}
string Human::get_nama()
{
    return this->nama;
}
void Human::set_nama(string nama)
{
    this->nama = nama;
}
string Human::get_jenis_kelamin()
{
    return this->jenis_kelamin;
}
void Human::set_jenis_kelamin(string jenis_kelamin)
{
    this->jenis_kelamin = jenis_kelamin;
}

// Destructor
Human::~Human()
{
}